src/SourceBuild/patches/fsharp/0001-Disable-errors-for-FS1182.patch

```diff
@@ -0,0 +1,22 @@
+From 909c664c5f1fe08a86176731296ffeebdf5c718a Mon Sep 17 00:00:00 2001
+From: Matt Thalman <mthalman@microsoft.com>
+Date: Tue, 14 Feb 2023 10:57:06 -0600
+Subject: [PATCH] Disable errors for FS1182
+
+---
```

Comment:
All patches should have a `Backport: <Issue/PR reference>` - e.g. https://github.com/dotnet/installer/blob/main/src/SourceBuild/patches/command-line-api/0001-Build-System.CommandLine.NamingConventionBinder-duri.patch#L7.  This makes patch maintenance easier.

Reply:
The issue/pr when fixed should remove the need for the patch.